devicemapper/thin_pool_watcher.go

```diff
@@ -85,16 +85,19 @@ func (w *ThinPoolWatcher) Stop() {
 }
 
 // GetUsage gets the cached usage value of the given device.
-func (w *ThinPoolWatcher) GetUsage(deviceId string) (uint64, error) {
+func (w *ThinPoolWatcher) GetUsage(deviceId string) uint64 {
 	w.lock.RLock()
 	defer w.lock.RUnlock()
 
 	v, ok := w.cache[deviceId]
 	if !ok {
-		return 0, fmt.Errorf("no cached value for usage of device %v", deviceId)
+		// TODO: ideally we should keep track of how many times we failed to get the usage for this
+		// device vs how many refreshes of the cache there have been, and return an error e.g. if we've
+		// had at least 1 refresh and we still can't find the device.
+		glog.V(5).Infof("no cached value for usage of device %v", deviceId)
```

Comment:
The problem with consuming the error here is we don't know whether the value returned is valid. For instance, is it a problem to set the `baseUsage` to 0 in handler.go when there isn't a cached value?